vendor/walletunlocker.proto

```diff
@@ -0,0 +1,338 @@
+}
+
+message GenSeedRequest {
+    /*
+    aezeed_passphrase is an optional user provided passphrase that will be used
+    to encrypt the generated aezeed cipher seed. When using REST, this field
+    must be encoded as base64.
+    */
+    bytes aezeed_passphrase = 1;
+
+    /*
+    seed_entropy is an optional 16-bytes generated via CSPRNG. If not
+    specified, then a fresh set of randomness will be used to create the seed.
+    When using REST, this field must be encoded as base64.
+    */
+    bytes seed_entropy = 2;
+}
+message GenSeedResponse {
+    /*
+    cipher_seed_mnemonic is a 24-word mnemonic that encodes a prior aezeed
+    cipher seed obtained by the user. This field is optional, as if not
+    provided, then the daemon will generate a new cipher seed for the user.
+    Otherwise, then the daemon will attempt to recover the wallet state linked
+    to this cipher seed.
+    */
+    repeated string cipher_seed_mnemonic = 1;
+
+    /*
+    enciphered_seed are the raw aezeed cipher seed bytes. This is the raw
+    cipher text before run through our mnemonic encoding scheme.
+    */
+    bytes enciphered_seed = 2;
+}
+
+message InitWalletRequest {
+    /*
+    wallet_password is the passphrase that should be used to encrypt the
+    wallet. This MUST be at least 8 chars in length. After creation, this
+    password is required to unlock the daemon. When using REST, this field
+    must be encoded as base64.
+    */
+    bytes wallet_password = 1;
+
+    /*
+    cipher_seed_mnemonic is a 24-word mnemonic that encodes a prior aezeed
+    cipher seed obtained by the user. This may have been generated by the
+    GenSeed method, or be an existing seed.
+    */
+    repeated string cipher_seed_mnemonic = 2;
+
+    /*
+    aezeed_passphrase is an optional user provided passphrase that will be used
+    to encrypt the generated aezeed cipher seed. When using REST, this field
+    must be encoded as base64.
+    */
+    bytes aezeed_passphrase = 3;
+
+    /*
+    recovery_window is an optional argument specifying the address lookahead
+    when restoring a wallet seed. The recovery window applies to each
+    individual branch of the BIP44 derivation paths. Supplying a recovery
+    window of zero indicates that no addresses should be recovered, such after
+    the first initialization of the wallet.
+    */
+    int32 recovery_window = 4;
+
+    /*
+    channel_backups is an optional argument that allows clients to recover the
+    settled funds within a set of channels. This should be populated if the
+    user was unable to close out all channels and sweep funds before partial or
+    total data loss occurred. If specified, then after on-chain recovery of
+    funds, lnd begin to carry out the data loss recovery protocol in order to
+    recover the funds in each channel from a remote force closed transaction.
+    */
+    ChanBackupSnapshot channel_backups = 5;
```

Comment:
Renaming to `wallet_unlocker` breaks because this is an import from `lnrpc`

Reply:
Here's some examples of how `walletkit.proto` handles the same situation:

`package walletrpc;`

`option go_package = "github.com/lightningnetwork/lnd/lnrpc/walletrpc";`

```
message ReleaseOutputRequest {
    // The unique ID that was used to lock the output.
    bytes id = 1;

    // The identifying outpoint of the output being released.
    lnrpc.OutPoint outpoint = 2;
}
```